src/store/region-context.tsx

```diff
@@ -60,12 +61,14 @@ export const RegionProvider = ({ children }: RegionProviderProps) => {
         residence: '',
     }
 
+    const qa_url_region = queryParams.get('region')?.toString()
+
     useEffect(() => {
         const is_eu_country_ip = eu_countries.includes(user_ip_country)
         const is_africa = african_countries.includes(user_ip_country)
         const is_eu_residence = eu_countries.includes(residence)
         const is_eu_location = is_eu_residence || (!residence && is_eu_country_ip)
-        const is_eu = isEuDomain()
+        const is_eu = isEuDomain() || eu_countries.includes(qa_url_region)
```

Comment:
`is_eu` is already declared in line 100, let's rename it here.

Reply:
It was a bad naming since past and not added by your PR but since we have a change on this line and getting SonarCloud code smell error. We should fix it here. Thanks Nuzhy.

Reply:
We will be needing to rename all the variables in the useEffect since all the variables are declared in line 98-105